blocks/carousel/carousel.js

```diff
@@ -100,8 +100,14 @@ export default function decorate(block) {
     [...row.children].forEach((col) => {
       const pic = col.querySelector('picture');
       if (pic) {
+        if (slideIndex === 1) {
+          pic.querySelector('img').setAttribute('loading', 'eager');
+        }
         slideContent.pic = pic;
         const img = pic.querySelector('img');
+        const ratio = (parseInt(img.height, 10) / parseInt(img.width, 10)) * 100;
+        slider.style.paddingBottom = `${ratio}%`;
+        slider.style.maxHeight = `${img.height}px`;
```

Comment:
This new ratio is creating a large space between the 2 areas of the carousel
![image](https://github.com/aemsites/888de/assets/3883795/1cdcff53-be32-49a0-b016-6bbee4504fc9)